### DISCUSSION TITLE
How to have visidata.pyobj.view() use custom visidatarc

### DISCUSSION THREAD

**Discussion by sourissouriante** [saulpw/visidata#2585] [Show and tell]
I might be doing something wrong, but when i explore python objects from within iypthon like this:

```
import visidata
visidata.pyobj.view(obj)
```

...then somehow my visidatarc isn't loaded. I might be the only one having this problem/demand or it might be trivial to solve for others. In any case, here is some recipe to resolve this. If I found something that can easily be improved/streamlined or if I can learn how I was doing something wrong, I'd be glad.

To load custom visidatarc in "ipython mode", do this:

Fire up longname command "pyobj-expr" (Ctrl+x) or "exec-python" and input "vd.loadConfigAndPlugins()"

Cheers!

**Comment by midichef** [comment]
If it helps, you can do the `loadConfigAndPlugins()` as a Python statement, like this:
```
import visidata
visidata.vd.loadConfigAndPlugins()
visidata.pyobj.view(obj)
```

**Reply by sourissouriante** [reply]
That's awesome, thanks for pointing out! Exactly what I strive to learn!
